jetty-server/src/main/config/etc/jetty-http.xml

```diff
@@ -34,6 +34,10 @@
         <Set name="idleTimeout"><Property name="jetty.http.idleTimeout" deprecated="http.timeout" default="30000"/></Set>
         <Set name="acceptorPriorityDelta"><Property name="jetty.http.acceptorPriorityDelta" deprecated="http.acceptorPriorityDelta" default="0"/></Set>
         <Set name="acceptQueueSize"><Property name="jetty.http.acceptQueueSize" deprecated="http.acceptQueueSize" default="0"/></Set>
+        <Set name="reuseAddress"><Property name="jetty.http.reuseAddress" default="true"/></Set>
+        <Set name="acceptedTcpNoDelay"><Property name="jetty.http.accepted.tcpNoDelay" default="true"/></Set>
+        <Set name="acceptedReceiveBufferSize"><Property name="jetty.http.accepted.receiveBufferSize" default="-1"/></Set>
+        <Set name="acceptedSendBufferSize"><Property name="jetty.http.accepted.sendBufferSize" default="-1"/></Set>
```

Comment:
Please:
* rename the property names to `jetty.http.acceptedTcpNoDelay` (i.e. not dot after `accepted`)
* add `acceptedReuseAddress` - use `reuseAddress` on the `ServerSocket` and `acceptedReuseAddress` on the accepted `Socket`

Reply:
I'm okay with renaming the properties.

Regarding `acceptedReuseAddress`, I don't think setting `SO_REUSEADDR` as any effect on an accepted socket.